pygmt/datasets/samples.py

```diff
@@ -272,14 +262,6 @@ def load_fractures_compilation(**kwargs):
         columns.
     """
 
-    if "suppress_warning" not in kwargs:
-        warnings.warn(
-            "This function has been deprecated since v0.6.0 and will be "
-            "removed in v0.9.0. Please use "
-            "load_sample_data(name='fractures') instead.",
-            category=FutureWarning,
-            stacklevel=2,
-        )
     fname = which("@fractures_06.txt", download="c")
     data = pd.read_csv(fname, header=None, sep=r"\s+", names=["azimuth", "length"])
```

Comment:
```suggestion
    data = pd.read_csv(fname, header=None, delim_whitespace=True, names=["azimuth", "length"])
```
Keeping this in line with what @seisman suggested in #2301.